iap/iap.js

```diff
@@ -1,59 +1,113 @@
-// Copyright (c) 2014 Intel Corporation. All rights reserved.
+// Copyright (c) 2015 Intel Corporation. All rights reserved.
```

Comment:
Please leave 2014 copyright, you only need to have new year copyright for new files.